_episodes_rmd/01-raster-structure.Rmd

```diff
@@ -113,8 +118,19 @@ library(rgdal)
 # render DSM for tutorial content background
 DSM_HARV <- raster("data/NEON-DS-Airborne-Remote-Sensing/HARV/DSM/HARV_dsmCrop.tif")
 
-# code output here - DEM rendered on the screen
-plot(DSM_HARV, main = "Continuous Elevation Map\n NEON Harvard Forest Field Site")
+library(ggplot2)
```

Comment:
@jsta  so i think we'd want library(ggplot2) in the load-libraries chunk. i tried to retrigger the build to see if it works when restarted.

Reply:
I think the chunk you are referencing is not echoed? My read is that the load-libraries chunk is correct given 21ebb9f.